tests/routes/articles.test.js

```diff
@@ -89,13 +96,20 @@ describe('Articles API', () => {
     it('should throw error if user not found', async () => {
       const response = await chai.request(app)
         .get(`${baseRoute}/articles/feed`)
-        .set('Authorization', `Bearer ${userToken}`);
+        .set('Authorization', `Bearer ${randomUserToken}`);
```

Comment:
Why was this changed? Why should a user be random?

Reply:
When I added a check if the user exists in the authentication middleware. Most tests failed because the "userToken" does not exist in the database so I changed them to randomUserToken.